File: exs/mundo_1/c++/005.cpp
/*

005 - Antecessor e sucessor
  Faça um programa que leia um número inteiro e mostre na tela o seu sucessor e
  seu antecessor

*/

#include <iostream>

using namespace std;

int main() {
  int numero, antecessor, sucessor;
  cin >> numero;

  antecessor = numero - 1;
  sucessor = numero + 1;

  cout << "O antecessor de " << numero << " é " << antecessor << "\n";
  cout << "O sucessor de " << numero << " é " << sucessor << "\n";


  return 0;
}

File: exs/mundo_1/javascript/005.js
/*
Desafio 005

Problema: Faça um programa que leia um número inteiro e mostre na tela o seu sucessor e seu antecessor

Resolução do problema:
*/

const input = require('readline-sync').question

let num = Number(input('Insira um número inteiro: '))
let intNum = Math.round(num)

console.log(`O numero digitado foi ${intNum}`)
console.log(`Seu antecessor é ${intNum - 1} e o sucessor ${intNum + 1}`)
